scripts/random_lifetime_extension.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import random
+import sys
+
+def generate_input(input_path, output_path, orig_lifetime, country):
+    """ This function takes a full CYCLUS input xml file,
+        finds the DeployInst block for country,
+        and extends the lifetime of LWR reactors 
+        according to a Gaussian distribution (mean 10, sd 3 [years])
+
+        Parameters
+        ----------
+        input_path: str
+            path for file to be copied and modified
+        output_path: str
+            path for the modified file
+        orig_lifetime: int
+            original lifetime for lwrs (to search for)
+        country: str
+            country (region) to make this change in
+        """
+    trig = False
+    trig2 = False
+    # these values are hard-coded for ease of use
```

Comment:
what values are hard-coded?

Reply:
actually they don't need to be anymore, since I use them in the ipynb. fixed!